integrations/drift/drift.go

```diff
@@ -90,7 +90,6 @@ func (api driftAPIProduction) request(method string, endpoint string, payload []
 	req.Header.Set("Content-Type", "application/json")
 	client := &http.Client{}
 	resp, err := client.Do(req)
-	defer resp.Body.Close()
```

Comment:
Ah, interesting. Can you check if there is a body, and close if if there is one?

It feels like this might or might not work, and become unreliable and hard to debug. Let me know your thoughts.